generators/app/prompts.js

```diff
@@ -216,3 +218,43 @@ function askModulesToBeInstalled(done, generator) {
         }
     );
 }
+
+function askForJavaVersion() {
+    if (this.existingProject) {
+        return;
+    }
+
+    const choices = [{ name: 'Java 11', value: '11' }, { name: 'Java 8', value: '1.8' }];
```

Comment:
We also support Java 12 (and may also support Java 13 soon), so we should probably either reword things a bit differently ("Java >= 11") or include other supported java versions.